mine/models/marts/fct_order_details.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+models:
+  - name: fct_orders_details
```

Comment:
Why not just fct_order_items or fct_sales? Details is not a bad name but it's not as precise as order_items. Sales is also sometimes preferred because it puts it in more "business" terms: a product was sold so it's a sale.